activestorage/app/models/active_storage/preview.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+# Some non-image blobs can be previewed: that is, they can be presented as images. A video blob can be previewed by
+# extracting its first frame, and a PDF blob can be previewed by extracting its first page.
+#
+# Active Storage provides previewers for videos and PDFs. You can add your own previewers or remove the built-in ones
+# by modifying +ActiveStorage.previewers+.
```

Comment:
Maybe need a quick note that order of previewers matter, as it's first match that'll be used.